lab_5_scrapper/scrapper.py

```diff
@@ -13,70 +62,116 @@ def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self.content = self._extract_config_content()
+        self._validate_config_content()
+        self._seed_urls = self.content.seed_urls
+        self._num_articles = self.content.total_articles
+        self._headers = self.content.headers
+        self._encoding = self.content.encoding
+        self._timeout = self.content.timeout
+        self._should_verify_certificate = self.content.should_verify_certificate
+        self._headless_mode = self.content.headless_mode
 
     def _extract_config_content(self) -> ConfigDTO:
         """
         Returns config values
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as f:
+            content = json.load(f)
+        return ConfigDTO(**content)
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters
         are not corrupt
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as f:
+            content = json.load(f)
+
+        seed_urls = content['seed_urls']
+        headers = content['headers']
+        total_articles_to_find_and_parse = content['total_articles_to_find_and_parse']
+        encoding = content['encoding']
+        timeout = content['timeout']
+        should_verify_certificate = content['should_verify_certificate']
+        headless_mode = content['headless_mode']
```

Comment:
is this the only way of checking all these fields? can you check fields without assigning them to variables and using config['smth'] directly?

Reply:
sdfghjk